src/common/fmt_common.h

```diff
@@ -38,11 +38,19 @@ template<class T>
 concept has_alt_fmt_print = requires(T t) {
   { t.alt_fmt_print(bool{}) } -> std::same_as<std::string>;
 };
+#if FMT_VERSION >= 110000
+template<class T>
+concept has_fmt_print_ctx = requires(
```

Comment:
Isn't this identical as the `#else` concept?

Reply:
No, there was a naming change.
FMT_VERSION >= 110000 uses `fmt::buffered_context` while else uses `fmt::buffer_context`.